src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -466,18 +466,20 @@ public void UpdateEnergyBalance(EnergyBalanceInfo energyBalance)
     /// <summary>
     ///   Updates the fluidity / rigidity slider tooltip
     /// </summary>
-    public void SetRigiditySliderTooltip(float rigidity)
+    public void SetRigiditySliderTooltip(int rigidity)
     {
         var healthChangeLabel = GetNode<Label>(RigiditySliderTooltipHealthLabelPath);
         var mobilityChangeLabel = GetNode<Label>(RigiditySliderTooltipSpeedLabelPath);
 
-        float healthChange = rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER;
-        float mobilityChange = -1 * rigidity * Constants.MEMBRANE_RIGIDITY_MOBILITY_MODIFIER;
+        float healthChange = rigidity * Constants.MEMBRANE_RIGIDITY_HITPOINTS_MODIFIER
+            / (float)Constants.MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO;
```

Comment:
This part should be done once, introduce like a new local `float convertedRigidity` that's used here and on line 477.